src/polymer-project.ts

```diff
@@ -107,8 +105,20 @@ export class PolymerProject {
 
   _splitFiles: Map<string, SplitFile> = new Map();
   _parts: Map<string, SplitFile> = new Map();
-  _analyzer: StreamAnalyzer;
-  _bundler: Bundler;
+
+  /**
+   * analyzer is a `Transform` that runs Hydrolysis analysis on the files, for
```

Comment:
You don't need the "analyze is" part. Just: "A `Transform`...". Same with bundler